tests/time_stream/test_infill.py

```diff
@@ -101,6 +101,10 @@ def test_get_with_invalid_type(self, get_input):
         (VARYING_GAPS, 1, (datetime(2025, 1, 3), datetime(2025, 1, 10)), False),
         (VARYING_GAPS, 2, (datetime(2025, 1, 6), datetime(2025, 1, 10)), False),
         (VARYING_GAPS, 3, datetime(2025, 1, 10), False),
+        (VARYING_GAPS, None, datetime(2025, 1, 3), True),
```

Comment:
Bit pedantic, but can the extra complete/insufficient tests be moved so they sit with their equivalents at the top

Also, for completeness shall we add a test where we have a gap, an interval and it returns true?